clients/admin-ui/src/pages/user-management/new-openid-user.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import NewUserForm from "user-management/NewUserForm";
+import UserManagementLayout from "user-management/UserManagementLayout";
+
+const CreateNewUser = () => (
+  <UserManagementLayout isNewOpenIDUser>
+    {/*
```

Comment:
Thanks for writing down the problems. This section looks a bit old so it may have some problems.
I think we shouldn't need a new page for open ID users. We can probably just have something on the Add User screen to select whether it's an Open ID user or not (only when Open ID is setup for it).